spec.html

```diff
@@ -40678,16 +40673,15 @@ <h1>
             _srcBuffer_: an ArrayBuffer or a SharedArrayBuffer,
             _srcByteOffset_: a non-negative integer,
             _srcLength_: a non-negative integer,
-            _cloneConstructor_: a constructor,
           ): either a normal completion containing an ArrayBuffer or an abrupt completion
         </h1>
         <dl class="header">
           <dt>description</dt>
           <dd>It creates a new ArrayBuffer whose data is a copy of _srcBuffer_'s data over the range starting at _srcByteOffset_ and continuing for _srcLength_ bytes.</dd>
         </dl>
         <emu-alg>
-          1. Let _targetBuffer_ be ? AllocateArrayBuffer(_cloneConstructor_, _srcLength_).
-          1. If IsDetachedBuffer(_srcBuffer_) is *true*, throw a *TypeError* exception.
+          1. Let _targetBuffer_ be ? <emu-meta suppress-effects="user-code">AllocateArrayBuffer(%ArrayBuffer%, _srcLength_)</emu-meta>.
+          1. Assert: IsDetachedBuffer(_surcBuffer_) is *false*.
```

Comment:
```suggestion
          1. Assert: IsDetachedBuffer(_srcBuffer_) is *false*.
```
Also should probably go on the previous line, since it's now an assertion about an argument. The fact that it isn't detached by the previous line is implied by the absence of user code on that line, I would think.

Reply:
Fixed typo and moved.